### PR TITLE
GH-795: Add high-res variants of toolbar icons

### PR DESCRIPTION
**Pull request by lbeurerkellner**
This PR adds high-resolution variants for a selection of the icons the N4JS IDE uses. In particular, this includes our toolbar icons so they do not look as blurry anymore (Fixes https://github.com/eclipse/n4js/issues/795).

Also, in order to avoid duplicates, I removed the existing icons from our `org.eclipse.n4js.n4mf.ui` bundle and delegated icon references of the N4JS Project Wizard to the `newprj_wiz.png` and `newprj_wizban.png` icons in the `org.eclipse.n4js.ui` bundle.

### CODE REVIEW

**Comment by mor-n4** [comment]
Now I want a new monitor!! 😈

**Comment by qtran-n4** [comment]
I want a HD curved monitor! 👿

File: plugins/org.eclipse.n4js.n4mf.ui/plugin.xml [hunk 1]
```diff
@@ -340,7 +340,7 @@ Contributors:
 		<handler 
 			class="org.eclipse.n4js.n4mf.ui.N4MFExecutableExtensionFactory:org.eclipse.xtext.ui.refactoring.ui.DefaultRenameElementHandler"
 			commandId="org.eclipse.xtext.ui.refactoring.RenameElement">
-			<activeWhen>
+			<activeWhen>
```
Comment (qtran-n4) [inline plugins/org.eclipse.n4js.n4mf.ui/plugin.xml]: trailing whitespace

File: plugins/org.eclipse.n4js.ui/icons/README.adoc [hunk 2]
```diff
@@ -110,10 +112,27 @@ show_hidden.gif                             - Composition of
 macrodef_obj.gif							- https://github.com/eclipse/eclipse.platform/blob/master/ant/org.eclipse.ant.ui/icons/full/obj16/macrodef_obj.png
 native_lib_path_attrib.gif					- https://github.com/eclipse/eclipse.jdt.ui/blob/master/org.eclipse.jdt.ui/icons/full/obj16/native_lib_path_attrib.png
 
+# wizards
```
Comment (qtran-n4) [inline plugins/org.eclipse.n4js.ui/icons/README.adoc]: `##` (two times 😈  :D)

**Comment by qtran-n4** [comment]
OK, you only changed a selected number if toolbar icons. Ignore the comment above then.

**Comment by yoosiba** [comment]
@qtran-n4 @lbeurerkellner please follow up and comments and make it mergable

**Pull request merged**